Triangulation_2/examples/Triangulation_2/terr_trian.cpp

```diff
@@ -5,17 +5,19 @@
 
 #include <CGAL/IO/Verbose_ostream.h>
 #include <CGAL/IO/OFF.h>
+#include <CGAL/boost/graph/IO/OFF.h>
 #include <CGAL/Projection_traits_xy_3.h>
 #include <CGAL/Triangulation_2.h>
 #include <CGAL/Delaunay_triangulation_2.h>
+#include <CGAL/boost/graph/graph_traits_Delaunay_triangulation_2.h>
+#include <CGAL/boost/graph/graph_traits_Triangulation_2.h>
```

Comment:
I would expect the Delaunay one to include the Triangulation one.

Reply:
Looking at the file I now see that we also have a variable of type `Triangulation_2` which explains the inclusion. So let's keep it.